modules/ecr/main.tf

```diff
@@ -1,8 +1,10 @@
-resource "aws_kms_key" "ecr_kms" {
+resource "aws_kms_key" "ecr_encryption" {
```

Comment:
The reason I've gone with this naming convention is because the resource name already tells us that it's a KMS key, so we don't have to specify that again. That said, I'm don't really mind what the resources are called in TF so happy to switch to this if you prefer

Reply:
**Done** (Ignored as agreed)